package.json

```diff
@@ -152,7 +152,7 @@
     "webpack": "gulp rev & webpack --progress --color --watch",
     "webpack-development": "gulp rev & webpack --progress --color",
     "webpack-stats": "webpack --profile --json",
-    "webpack-production": "NODE_ENV=production gulp rev & webpack --color --config webpack.production.config.js",
+    "webpack-production": "gulp rev & node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js --progress --color --config webpack.production.config.js",
```

Comment:
can we not add `--progress`, it makes the output of our build really annoying to read